rpc/clients/typescript/package.json

```diff
@@ -37,8 +30,6 @@
     },
     "homepage": "https://github.com/0xProject/0x-mesh/rpc/clients/typescript/README.md",
     "dependencies": {
-        "@0x/assert": "^2.1.2",
```

Comment:
The `@0x/assert` package is used in `ws_client.ts`, and is therefore not a `devDep`.